bucket/xnviewmp.json

```diff
@@ -14,9 +14,13 @@
         }
     },
     "extract_dir": "XnViewMP",
-    "pre_install": "New-Item \"$persist_dir\\xnview.ini\" -ErrorAction SilentlyContinue",
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\xnview.ini\") -or !(Get-Item \"$persist_dir\\xnview.ini\").Length) {",
+        "   New-Item \"$persist_dir\\xnview.ini\" -ItemType File -Force | Out-Null",
```

Comment:
```suggestion
        "   New-Item \"$dir\\xnview.ini\" -ItemType File -Force | Out-Null",
```

Reply:
Whether we create `xnview.ini` in the perist_dir or dir doesn't matter. `"persist": "xnview.ini"` takes care of linking it to dir anyways. So it's just a matter of style.

Reply:
@CrendKing I added more details in the comment. Please check if this explains or not.

Reply:
You are correct. However, the `pre_install` script happens between (1) and (2), which guarantees a file named "xnview.ini" exists in persist_dir with the PR script. Since the file exists, (2) will always happen and the first part of (4) will never happen. Right?

Reply:
Your script works fine, but I think this is easier to understand (and maintain):
```
    "pre_install": [
        "# --- If xnview.ini exists as a folder, then remove it ---",
        "if (Test-Path \"$persist_dir\\xnview.ini\" -PathType Container) {",
        "    Remove-Item \"$persist_dir\\xnview.ini\" -Force -Recurse",
        "}",
        "if (!(Test-Path \"$persist_dir\\xnview.ini\")) {",
        "   New-Item \"$dir\\xnview.ini\" -ItemType File | Out-Null",
        "}"
    ],
```

If there is something wrong, please tell me.

Reply:
If you guys want to fix the xnview.ini directory situation, feel free to do that.

The `!(Get-Item \"$persist_dir\\xnview.ini\").Length` check is indeed not needed for this PR. I'll remove.